Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psm1

```diff
@@ -367,10 +367,13 @@ function Resize-VmfsVolume {
 
     .PARAMETER ClusterName
      Cluster name
-    
+
     .PARAMETER DeviceNaaId
      NAA ID of device associated with the existing VMFS volume
 
+    .PARAMETER DatastoreName
+     Datastore name
```

Comment:
Could this description be updated to specify that this parameter is optional, and the value is used to update the name of a snapshot datastore?

Reply:
Updated the comment